pkgdb2/templates/list_actions.html

```diff
@@ -123,7 +123,8 @@
         </a>
       </td>
       <td>
-        {% if action.package %}
+        {% if action.package or (request.action == 'request.package' and
```

Comment:
What is ``request`` here? (that you call in ``request.action`` and ``request.status``)